app/controllers/application_controller.rb

```diff
@@ -6,14 +6,14 @@ class ApplicationController < ActionController::Base
   # run these methods on page load
   before_filter :current_user
 
-  private
- 
-  # Finds the User with the ID stored in the session with the key
-  # :current_user_id This is a common way to handle user login in
-  # a Rails application; logging in sets the session value and
-  # logging out removes it.
+  helper_method :current_user
```

Comment:
I mean do you really need this in the view anyway? Why bother making it a helper_method?